openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -15,6 +15,63 @@ function update_len() {
     $('#excerpts-excerpt-len').html(2000 - len).css('color', color);
 }
 
+/**
+ * This is needed because jQuery has no forEach equivalent that works with jQuery elements instead of DOM elements
+ * @param selector - css selector used by jQuery
+ * @returns {*[]} - array of jQuery elements
+ */
+function getJqueryElements(selector){
+    const queryResult = $(selector);
+    return Array.from(queryResult).map((ele,index) => queryResult.eq(index));
```

Comment:
Array.from is ES6 so not supported in IE11. The transpiler will allow ES6 syntax but will not polyfill Array.from